tests/by-util/test_cp.rs

```diff
@@ -444,16 +444,26 @@ fn test_cp_arg_interactive() {
 
 #[test]
 #[cfg(not(target_os = "android"))]
-fn test_cp_arg_interactive_update() {
-    // -u -i won't show the prompt to validate the override or not
-    // Therefore, the error code will be 0
+fn test_cp_arg_interactive_verbose() {
     let (at, mut ucmd) = at_and_ucmd!();
     at.touch("a");
     at.touch("b");
-    ucmd.args(&["-i", "-u", "a", "b"])
+    ucmd.args(&["-vi", "a", "b"])
         .pipe_in("N\n")
-        .succeeds()
-        .no_stdout();
+        .fails()
+        .stdout_is("skipped 'b'\n");
+}
```

Comment:
What's the reason for replacing the `-iu`-test with a `-vi`-test?

Reply:
typo, let me fix that :)